docs/src/codegen.md

```diff
@@ -222,26 +324,27 @@ playwright codegen --load-storage=auth.json github.com/microsoft/playwright
 pwsh bin/Debug/netX/playwright.ps1 codegen --load-storage=auth.json github.com/microsoft/playwright
 ```
 
-<img width="1261" alt="Screenshot 2022-08-03 at 13 33 40" src="https://user-images.githubusercontent.com/13063165/182599680-05297b4e-c258-4416-8daa-b8637c1db120.png" />
+######
+* langs: js
 
-Use the `open` command with `--load-storage` to open the saved `auth.json`.
+<img width="1394" alt="github signed in showing use of load storage js" src="https://user-images.githubusercontent.com/13063165/220555719-b94dc157-688b-45bd-b61a-282edb4b0d16.png" />
```

Comment:
lets retake the next 4 screenshots without the security announcement (click on x there).